### PR TITLE
Add patch release v3.6.1 with fix to k8s apiserver

### PR DESCRIPTION
**Pull request by tuommaki**
Kubernetes apiserver has a memory leak in v1.11.1. This can be mitigated
by applying a fix from apimachinery, but this is not merged in upstream
v1.11.x release yet. There is a custom built GiantSwarm release for this
(as AMD64 hyperkube image) that can be used while waiting for new
upstream release.

Towards https://github.com/giantswarm/giantswarm/issues/4187

### CODE REVIEW

File: v_3_6_1/worker_template.go [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+      Description=k8s-kubelet
+      StartLimitIntervalSec=0
+
+      [Service]
+      TimeoutStartSec=300
+      Restart=always
+      RestartSec=0
+      TimeoutStopSec=10
+      EnvironmentFile=/etc/network-environment
+      Environment="IMAGE={{ .RegistryDomain }}/giantswarm/hyperkube:v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm"
```
Comment (tuommaki) [inline v_3_6_1/worker_template.go]: This is the actual change to `v3.6.0` version.

File: v_3_6_1/master_template.go [hunk 2]
```diff
@@ -0,0 +1,2297 @@
...
+          tolerations:
+          - key: node-role.kubernetes.io/master
+            operator: Exists
+            effect: NoSchedule
+          hostNetwork: true
+          priorityClassName: system-node-critical
+          serviceAccountName: kube-proxy
+          containers:
+            - name: kube-proxy
+              image: {{ .RegistryDomain }}/giantswarm/hyperkube:v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm
```
Comment (tuommaki) [inline v_3_6_1/master_template.go]: This is the actual change to `v3.6.0` version.

File: v_3_6_1/master_template.go [hunk 3]
```diff
@@ -0,0 +1,2297 @@
...
+      name: k8s-api-server
+      namespace: kube-system
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - name: k8s-api-server
+        image: {{ .RegistryDomain }}/giantswarm/hyperkube:v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm
```
Comment (tuommaki) [inline v_3_6_1/master_template.go]: This is the actual change to `v3.6.0` version.

File: v_3_6_1/master_template.go [hunk 4]
```diff
@@ -0,0 +1,2297 @@
...
+      name: k8s-controller-manager
+      namespace: kube-system
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - name: k8s-controller-manager
+        image: {{ .RegistryDomain }}/giantswarm/hyperkube:v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm
```
Comment (tuommaki) [inline v_3_6_1/master_template.go]: This is the actual change to `v3.6.0` version.

File: v_3_6_1/master_template.go [hunk 5]
```diff
@@ -0,0 +1,2297 @@
...
+      name: k8s-scheduler
+      namespace: kube-system
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - name: k8s-scheduler
+        image: {{ .RegistryDomain }}/giantswarm/hyperkube:v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm
```
Comment (tuommaki) [inline v_3_6_1/master_template.go]: This is the actual change to `v3.6.0` version.

File: v_3_6_1/master_template.go [hunk 6]
```diff
@@ -0,0 +1,2297 @@
...
+      ExecStopPost=-/usr/bin/docker rm -f $NAME
+  - name: etcd2.service
+    command: stop
+    enable: false
+    mask: true
+  - name: etcd3.service
+    enable: true
+    command: start
+    content: |
+
```
Comment (tuommaki) [inline v_3_6_1/master_template.go]: This is the actual change to `v3.6.0` version.

**Comment by xh3b4sd** [comment]
Is this the copy or all new including change? I cannot tell it apart.

**Comment by r7vme** [comment]
@tuommaki please do like that:
1. PR1. Just a copy paste (no files changes)
2. PR2. Changes version/package name for new version, adds blank changelog
3. PR3. Your actual changes.

PR1 and PR2 can be merged w/o review.

PR3 regular review process.

**Comment by tuommaki** [comment]
Ok. Redoing this.

**Pull request closed without merging**